src/ixdat/techniques/ms.py

```diff
@@ -3,50 +3,19 @@
 import re
 import numpy as np
 
-# FIXME: This is for MSCalibration.export, but shouldn't have to be here.
-import json
-import warnings
-
-from ..measurements import Measurement, Calibration
+from ..measurements import Measurement
 from ..spectra import Spectrum, SpectrumSeries, SpectroMeasurement
 from ..plotters import MSPlotter, MSSpectroPlotter
 from ..plotters.ms_plotter import STANDARD_COLORS
 from ..exporters import MSExporter, MSSpectroExporter
-from ..exceptions import QuantificationError
-from ..constants import (
-    AVOGADRO_CONSTANT,
-    BOLTZMANN_CONSTANT,
-    STANDARD_TEMPERATURE,
-    STANDARD_PRESSURE,
-    DYNAMIC_VISCOSITIES,
-    MOLECULAR_DIAMETERS,
-    MOLAR_MASSES,
-)
-from ..data_series import ValueSeries
-from ..db import Saveable
 from ..tools import deprecate
 from ..config import plugins
-
-
-def _with_siq_quantifier(method):
```

Comment:
This is handled within the `Calculator` now, I assume?

Reply:
Right. By inheriting from `Calculator`, a `siqCalculator` also inherits from `Saveable` and gets put into a `MemoryBackend` to get correctly connected to a new measurement made by e.g. `copy()` , `cut()`, or `select()` :)
Homemade `sqlalchemy` at work!